addons/true-youtube-links/userscript.js

```diff
@@ -0,0 +1,11 @@
+﻿export default async function ({ addon, global, console }) {
+  addon.settings.addEventListener("change", () => console.log("changed!"));
+  while (true) {
```

Comment:
Why a loop? Posts can't be dynamically added without the page refreshing. You should use document.querySelectorAll then loop through the elements.

Reply:
Comments can.

Reply:
Oh, ok. You should use document.querySelectorAll anyway. You can keep the while true loop (and remove the second line of the file that makes a useless console log)

Reply:
If when you first load the page (the first time the while loop runs) there's more than one YouTube link, not using querySelectorAll will only match the first link.

Reply:
But one loop later it also replaces the next one.
It really replaces every link. I checked that.

Reply:
Hmm, you're right